google_compute_engine/config_manager.py

```diff
@@ -101,7 +102,7 @@ def WriteConfig(self, config_file=None):
     """
     config_file = config_file or self.config_file
     config_name = os.path.splitext(os.path.basename(config_file))[0]
-    config_lock = '/var/lock/google_%s.lock' % config_name
+    config_lock = constants.LOCALSTATEDIR + '/lock/google_%s.lock' % config_name
```

Comment:
In this case where you have a string with multiple variables, its preferable to use:
```
config_lock = ( 
    '%s/lock/google_%s.lock' % (constants.LOCALSTATEDIR, config_name))
```